src/middlewared/middlewared/event.py

```diff
@@ -28,7 +33,7 @@ def register(self, name, description, private, returns, no_auth_required, no_aut
         if private:
             self.__events_private.add(name)
 
-    def get_event(self, name):
+    def get_event(self, name:str) -> dict[str]:
```

Comment:
This can be null as well so we should mark it as such i.e `typing.Optional`

Reply:
Also this would be `typing.Dict[key_type, value_type]`

Reply:
Please see different modules in middleware which already have type hinting in place, that would be a nice point to see how we have been doing it so far